sys/aarch64/pmap.c

```diff
@@ -286,7 +294,7 @@ void pmap_kenter(vaddr_t va, paddr_t pa, vm_prot_t prot, unsigned flags) {
 
   klog("Enter unmanaged mapping from %p to %p", va, pa);
 
-  pte_t pte = make_pte(pa, prot, flags);
+  pte_t pte = make_pte(pa, vm_prot_map[prot], flags);
```

Comment:
Why cannot we do it the way we did it earlier? To take the appropriate prots in the make_pte function?